src/app/header/model-header.component.ts

```diff
@@ -0,0 +1,147 @@
+import { Component, OnInit, OnDestroy } from "@angular/core";
```

Comment:
Make icon bigger to make it visible correctly when mouse is hovered on.

Reply:
fixed